testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -573,34 +573,32 @@ def test_sklearn_affinity_propagation(self, ens1):
                               "clusters: {0} {1}".format(len(cc1), len(cc2))
 
     def test_ClusterCollection_init(self, cc):
-        if np.testing.assert_allclose(cc.clusters[0].elements , [0,1,2])and \
-               np.testing.assert_allclose(cc.clusters[1].elements , [3, 4]) \
-                   and \
-               np.testing.assert_allclose(cc.clusters[2].elements , [5, 6, 7])\
-                   and \
-               cc.clusters[0].centroid == 1 and \
-               cc.clusters[1].centroid == 3 and \
-               cc.clusters[2].centroid == 5: \
-                      print("ClusterCollection was not constructed correctly")
+        assert_equal(cc.clusters[0].elements , [0,1,2]) and \
+        assert_equal(cc.clusters[1].elements , [3, 4]) and \
+        assert_equal(cc.clusters[2].elements , [5, 8, 7]) and \
+        assert_equal(cc.clusters[0].centroid , 1) and \
+        assert_equal(cc.clusters[1].centroid , 3) and \
+        assert_equal(cc.clusters[2].centroid , 5), \
+                   "ClusterCollection was not constructed correctly"
 
     def test_Cluster_init(self, cluster):
-        if np.testing.assert_allclose(cluster.elements,[0, 1, 2]) and \
-               cluster.centroid == 1: \
-                      print("Cluster was not constructed correctly")
+        assert_equal(cluster.elements,[0, 1, 2]) and \
+              assert_equal(cluster.centroid , 1), \
+                      "Cluster was not constructed correctly"
 
     def test_ClusterCollection_get_ids(self, cc):
-        assert cc.get_ids() == [0, 1, 2], \
-                     "ClusterCollection ids aren't as expected"
+        assert_equal(cc.get_ids() , [0, 1, 2], \
+                     err_msg="ClusterCollection ids aren't as expected")
 
     def test_ClusterCollection_get_centroids(self, cc):
-        assert cc.get_centroids() == [1, 3, 5], \
-                     "ClusterCollection centroids aren't as expected"
+        assert_equal(cc.get_centroids() , [1, 3, 5], \
+                     err_msg="ClusterCollection centroids aren't as expected")
 
     def test_cluster_add_metadata(self, cluster):
         metadata = cluster.elements*10
         cluster.add_metadata('test', metadata)
-        assert np.all(cluster.metadata['test'] == metadata), \
-                     "Cluster metadata isn't as expected"
+        assert_equal(cluster.metadata['test'] , metadata, \
```

Comment:
```suggestion
        assert_equal(cluster.metadata['test'], metadata, \
```

Reply:
To add to @PicoCentauri's comment - `\` are highly discouraged in PEP8. Where possible you should use implicit continuation. Indeed here the `\` is unecessary because you  have the `)` at the end.
```suggestion
        assert_equal(cluster.metadata['test'], metadata,
```